deployments/container/reconfigure-mig.sh

```diff
@@ -525,27 +541,36 @@ if [ "${?}" != "0" ]; then
 fi
 
 if [ "${CDI_ENABLED}" = "true" ]; then
-	echo "Running nvidia-smi"
-	chroot ${DRIVER_ROOT_CTR_PATH} nvidia-smi >/dev/null
-	if [ "${?}" != "0" ]; then
-		exit_failed
+	if [ "${DRIVER_ROOT_CTR_PATH}" = "/host" ]; then
+		echo "Running nvidia-smi"
+		chroot ${DRIVER_ROOT_CTR_PATH} nvidia-smi >/dev/null
+		if [ "${?}" != "0" ]; then
+			exit_failed
+		fi
 	fi
 
 	echo "Creating NVIDIA control device nodes"
-	nvidia-ctk system create-device-nodes --control-devices --driver-root=${DRIVER_ROOT_CTR_PATH}
+	nvidia-ctk system create-device-nodes \
+		--control-devices \
+		--driver-root=${DEV_ROOT_CTR_PATH}
```

Comment:
Do we want to merge https://github.com/NVIDIA/nvidia-container-toolkit/pull/526 first and switch to `--dev-root` here?

Reply:
Updated to use `--dev-root`.